docs/product-installation-guide/src/main/asciidoc/clustering-ps-configure-domain-proc.adoc

```diff
@@ -71,7 +103,15 @@ ifdef::PAM[]
 </xa-datasource>
 ----
 endif::PAM[]
-. In the `security-setting` element of the `activemq` subsystem in the `"full-ha"` profile, add the `kie-server` role:
++
+[WARNING]
+====
+You must use two different databases for {KIE_SERVER} runtime data and EJB timer data.
+====
++
+.  In the `<subsystem xmlns="urn:jboss:domain:messaging-activemq:2.0">` element, make the following changes:
+.. Replace `<cluster password="${jboss.messaging.cluster.password:CHANGE ME!!}"/>` with `<cluster user="<CLUSTER_USER>" password="<CLUSTER_PWD>"/>` where `<CLUSTER_USER>` is the user used by EAP to form a messaging cluster and `<CLUSTER_PWD>` is the password for that user.
```

Comment:
"used by {EAP}". I'd search the doc in atom to see if there are any other hard-coded EAPs. They should all be the attribute. I don't think we should go any shorter than what the attribute renders as. If you find that's too cumbersome, I trust your judgment.